js/client-rooms.js

```diff
@@ -80,20 +80,24 @@
 				app.roomsFirstOpen = 2;
 			}
 		},
+
 		renderRoomBtn: function (roomData) {
 			var id = toID(roomData.title);
-			var buf = '<div><a href="' + app.root + id + '" class="ilink"><small style="float:right">(' + Number(roomData.userCount) + ' users)</small><strong><i class="fa fa-comment-o"></i> ' + BattleLog.escapeHTML(roomData.title) + '<br /></strong><small>' + BattleLog.escapeHTML(roomData.desc || '');
+			if (!(roomData.subRooms && roomData.subRooms.length))
+				var buf = '<div class="ilink" style="margin-bottom:7px"><a style="color:inherit" href="' + app.root + id + '"><small style="float:right">(' + Number(roomData.userCount) + ' users)</small><strong><i class="fa fa-comment-o"></i> ' + BattleLog.escapeHTML(roomData.title) + '<br /></strong><small>' + BattleLog.escapeHTML(roomData.desc || '') + '</a></div>';
 			if (roomData.subRooms && roomData.subRooms.length) {
-				buf += '<br/><i class="fa fa-level-up fa-rotate-90"></i> Subrooms: <strong>';
+				buf = '<div class="ilink" style="margin-bottom:-2px"><a style="color:inherit" href="' + app.root + id + '"><small style="float:right">(' + Number(roomData.userCount) + ' users)</small><strong><i class="fa fa-comment-o"></i> ' + BattleLog.escapeHTML(roomData.title) + '<br /></strong><small>' + BattleLog.escapeHTML(roomData.desc || '') + '<br><i>Subrooms: </i></a><strong></div>';
```

Comment:
This is not valid HTML. You're missing a `</small>`, you have an extraneous `<strong>`, etc.